src/tests/functional/inference_engine/transformations/common_optimizations/transpose_sinking_binary_test.cpp

```diff
@@ -0,0 +1,1134 @@
+// Copyright (C) 2022 Intel Corporationstd
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <transformations/common_optimizations/transpose_sinking_binary.hpp>
```

Comment:
I think we need to rename files `transpose_sinking_binary`/`transpose_sinking_binary_test` to reflect concat and split handling